Canvas/CanvasUITests/Dashboard/DashboardTests.swift

```diff
@@ -80,6 +80,26 @@ enum CourseDetails {
 class DashboardTests: CanvasUITests {
     override var user: User? { return .student1 }
 
+    func testAnnouncementBelowInvite() {
+        CourseInvitation.acceptButton(id: "998").waitToExist()
+        GlobalAnnouncement.toggle(id: "2").waitToExist()
+        XCTAssertLessThan(CourseInvitation.acceptButton(id: "998").frame.maxY, GlobalAnnouncement.toggle(id: "2").frame.minY)
```

Comment:
@GabrielPalmer Check out @vanwagonet's use of `frame` here. Might come in handy.